inventory-service/pom.xml

```diff
@@ -22,6 +22,7 @@
         <java.version>21</java.version>
         <spring-cloud.version>2024.0.0</spring-cloud.version>
         <springdoc-openapi.version>2.8.3</springdoc-openapi.version>
+        <jooq.version>3.19.18</jooq.version>
```

Comment:
@coderabbitai, can you please create an issue to revert this line when spring boot supports in dependency management